src/test/java/automationpractice/com/pages/AddressCheckoutPage.java

```diff
@@ -0,0 +1,79 @@
+    private WebElementFacade clickOnProceedToCheckoutButtonInAddressLabel;
+
+    @FindBy(xpath = "//*[contains(@class,'address_add submit')]//*[contains(@class,'button')]")
+    private WebElementFacade clickAddANewAddressButton;
+
+    @FindBy(xpath = "//*[@id='address_delivery']//*[@class='address_firstname address_lastname']")
+    private WebElementFacade nameAndLastNameInDeliveryAddress;
+
+    @FindBy(xpath = "//*[@id='address_delivery']//*[@class='address_city address_state_name address_postcode']")
+    private WebElementFacade cityAndStateAndZipCodeInDeliveryAddress;
+
+    @FindBy(xpath = "//*[@id='center_column']//*[@class='checkbox addressesAreEquals']")
+    private WebElementFacade clickDisagreeUseTheDeliveryAddressAsTheBillingAddress;
+
+    @FindBy(xpath = "//*[@id='address_invoice']//*[@class='address_firstname address_lastname']")
+    private WebElementFacade nameAndLastNameInBillingAddress;
+
+    @FindBy(xpath = "//*[@id='address_invoice']//*[@class='address_city address_state_name address_postcode']")
+    private WebElementFacade cityAndStateAndZipCodeInBillingAddress;
+
+    @FindBy(xpath = "//*[@id='ordermsg']//*[@class='form-control']")
+    private WebElementFacade addCommentInAddressTab;
+    public void clickOnProceedToCheckoutButtonInAddressLabel() {
+        clickOnProceedToCheckoutButtonInAddressLabel.click();
+    }
+
+    private NewAddressPage newAddressPage;
+
+    public void clickAddANewAddressButton() {
+        clickAddANewAddressButton.click();
+    }
+
+    public void selectBillingAddressFromList(NewAddressData addressData) {
+        clickDisagreeUseTheDeliveryAddressAsTheBillingAddress.click();
+        waitABit(1000);
+        find(By.xpath("//*[@id='id_address_invoice']//*[contains(text(), '" + addressData.getNewAddressName() + "')]")).click();
+    }
+
+    public void selectDeliveryAddressFromList(NewAddressData addressData) {
+        find(By.xpath("//*[@id='id_address_delivery']//*[contains(text(), '" + addressData.getNewAddressName() + "')]")).click();
+    }
+
+    public void selectDeliveryAndBillingAddressFromList(NewAddressData addressData) {
+        selectDeliveryAddressFromList(addressData);
+        selectBillingAddressFromList(addressData);
+    }
+
+    public String getNameAndLastNameInDeliveryInAddressCheckoutPage() {
+        return nameAndLastNameInDeliveryAddress.getText();
+    }
+
+    public String getCityAndStateAndZipCodeInDeliveryInAddressCheckoutPage() {
+        return cityAndStateAndZipCodeInDeliveryAddress.getText();
+    }
+
+    public String getNameAndLastNameInBillingInAddressCheckoutPage() {
+        return nameAndLastNameInBillingAddress.getText();
+    }
+
+    public String getCityAndStateAndZipCodeInBillingInAddressCheckoutPage() {
+        return cityAndStateAndZipCodeInBillingAddress.getText();
+    }
+
+    public void addCommentInAddressTab() {
+        addCommentInAddressTab.type("entrance from the yard!");
+    }
+
+}
```

Comment:
Nazwy metod. Jeśli stworzyłeś te metody w klasie AddressCheckoutPage to nie dodawaj na koniec każdej nazwy metody "InAddressCheckoutPage" albo "InAddressTab", to jest jasne, że to jest tutaj